src/components/SearchForm/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { AiOutlineSearch } from 'react-icons/ai';
+
+import styles from './index.module.css';
+
+function SearchForm({ fetchGitRepo }) {
+    const [username, setUsername] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        fetchGitRepo(username);
+        setUsername('');
+    };
+
+    const updateInput = (e) => {
+        const input = e.target.value;
```

Comment:
In the controlled form, I don't think you want to set a variable to the input value
The controlled form should have another useState for the input variable. 
(Not sure how important this is as clearly it functions correctly)

Reply:
Yes, it seems to be fine. I wondered about that as well (having another state) but I think we'll leave it as it is for now, it seems to grasp the value exactly at the right time and update the state with that value.